ocs_ci/ocs/ui/infra_ui.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version
+
+
+logger = logging.getLogger(__name__)
+
+
+class InfraUI(PageNavigator):
```

Comment:
the class name does not reflect its purpose that is mentioned in the docstring below